src/transformers/models/xglm/modeling_tf_xglm.py

```diff
@@ -463,19 +463,12 @@ def _prepare_decoder_attention_mask(
     ) -> tf.Tensor:
         # create causal mask
         # [bsz, seq_len] -> [bsz, 1, tgt_seq_len, src_seq_len]
-        combined_attention_mask: tf.Tensor | None = None
-        if input_shape[-1] > 1:
-            combined_attention_mask = _make_causal_mask(input_shape, past_key_values_length)
-
-        if attention_mask is not None:
+        combined_attention_mask = _make_causal_mask(input_shape, past_key_values_length)
+        if attention_mask is None:
+            return combined_attention_mask
+        else:
```

Comment:
ultra nit: if we return in an if statement, we don't need the `else`

Reply:
Fixed!